### PR TITLE
16 commons version upgrade

### PR DESCRIPTION
**Pull request by slifty**
The three commons libraries identified in #16 were all pretty straightforward upgrades; no code needed to be changed for our tests to pass.

### CODE REVIEW

File: psm-app/build.gradle [hunk 1]
```diff
@@ -23,7 +23,7 @@ ext.libs = [
     commons_codec: 'commons-codec:commons-codec:1.11',
     commons_fileupload: 'commons-fileupload:commons-fileupload:1.3.3',
     commons_io: 'commons-io:commons-io:2.6',
-    commons_lang: 'commons-lang:commons-lang:2.4',
+    commons_lang: 'org.apache.commons:commons-lang3:3.7',
```
Comment (jasonaowen) [inline psm-app/build.gradle]: This is a different artifact, not just a different version of the same artifact. We still have code importing from `commons-lang:2.4`, which is provided through a transitive dependency of `velocity`.

Separately, we were previously bringing in `commons-lang3:3.5` through `hapi-fhir-client-okhttp`; [I added code that depends on `commons-lang3`](https://github.com/OpenTechStrategies/psm/commit/2787bdc7468e2cb810035807c093d27b293c38fb#diff-57e3bbf3fd0939212678b85ec47ac0bdR7) without explicitly declaring it as a dependency, which was incorrect.

So, we should add `commons-lang3` as a dependency, optionally upgrade `commons-lang` to its latest ([2.6](https://search.maven.org/#artifactdetails%7Ccommons-lang%7Ccommons-lang%7C2.6%7Cjar)), and migrate our code to `commons-lang3`.

**Comment by jasonaowen** [comment]
The LEIE interaction still works! As hoped/expected. :tada:

File: psm-app/build.gradle (hunk 1 above)
Comment (slifty) [reply psm-app/build.gradle]: @jasonaowen makes total sense, I'll update to 2.6 and add lang3 -- (2.6 worked when I tested it as well)

Comment (cecilia-donnelly) [reply psm-app/build.gradle]: Thanks you two!

**Pull request merged**